tests/functional/ext/docparams/raise/missing_raises_doc_exc_inheritance.py

```diff
@@ -0,0 +1,45 @@
+"""Tests for missing-raises-doc for class inheritance."""
+# pylint: disable=missing-class-docstring
+
+class CustomError(NameError):
+    pass
+
+
+class CustomChildError(CustomError):
+    pass
+
+
+def test_find_missing_raise_for_parent():
```

Comment:
We have an unexpected ``missing-raises-doc`` here. It's something that really need to be fixed as we have documentation for the raises line 15/16.

Reply:
Why does the tests/functional/ext/docparams/raise/missing_raises_doc_exc_inheritance.txt not work?

Reply:
I'm not sure, the way ``missing-raises-doc`` raise messages probably changed ?

Reply:
forgot a code comment...